code/modules/clothing/ears/ears.dm

```diff
@@ -4,7 +4,19 @@
 	icon_state = "earmuffs"
 	item_state = "earmuffs"
 	flags = EARBANGPROTECT
-	flags_2 = HEALS_EARS_2
 	strip_delay = 15
 	put_on_delay = 25
 	resistance_flags = FLAMMABLE
+
+/obj/item/clothing/ears/earmuffs/ComponentInitialize()
```

Comment:
I thought `ComponentInitialize` was being deprecated

Reply:
Not that I know of?